README.md

```diff
@@ -5,38 +5,41 @@
 
 The Adobe Experience Platform Edge Network mobile extension allows you to send data to the Adobe  Edge Network from a mobile application. This extension allows you to implement Adobe Experience Cloud capabilities in a more robust way, serve multiple Adobe solutions though one network call, and simultaneously forward this information to the Adobe Experience Platform.
 
-The AEP Edge Network mobile extension is an extension for the [Adobe Experience Platform SDK](https://aep-sdks.gitbook.io) and requires the `AEPCore` and `AEPServices` extensions for event handling, as well as the `AEPEdgeIdentity` extension for retrieving the identities, such as ECID.
+The AEP Edge Network mobile extension is an extension for the Adobe Experience Platform SDK and requires the `AEPCore` and `AEPServices` extensions for event handling, as well as the `AEPEdgeIdentity` extension for handling identities, such as `ECID`.
 
-To learn more about this extension, read the [Adobe Experience Platform Edge Network](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension) documentation.
+## Documentation
+
+To learn more about this extension, see the [Adobe Experience Platform Edge Network](https://developer.adobe.com/client-sdks/documentation/edge-network/) documentation.
+
+Additional information about SDK usage and architecture can be found in the [Documentation](Documentation) directory.
 
+## Getting started
 ### Installation
 
 Integrate the Edge Network extension into your app by including the following in your gradle file's `dependencies`:
 
 ```gradle
-implementation 'com.adobe.marketing.mobile:edgeidentity:1.+'
-implementation 'com.adobe.marketing.mobile:edge:1.+'
-implementation 'com.adobe.marketing.mobile:core:1.+'
+implementation 'com.adobe.marketing.mobile:core:2.0.0'
```

Comment:
We will point this to getting started page.

Reply:
Integrate the Edge Network extension into your app by including the following the [getting started guide](Documentation/getting-started.md).

Reply:
Are you going to update getting started to 2+ in this PR?
https://github.com/timkimadobe/aepsdk-edge-android/blob/update-tutorial-2.0/Documentation/getting-started.md#add-the-edge-network-extension-to-your-app

Reply:
You can wait for edgeIdentity doc PR reviewed then change.